### DISCUSSION TITLE
Please make Projects a top-level navigation item

### DISCUSSION THREAD

**Discussion by chrysb** [community/community#12066] [Projects and Issues]
Right now it takes 4 clicks for me to get back to Projects, which is supposed to be my command center.

1. Click home
2. Click a respository
3. Click the Projects tab
4. Click project

**Comment by MarkBennett** [comment]
Totally agree on this! At the very least it would be awesome to see this integrate with the new Command Pallette and keyboard shortcuts so I can quickly navigate to my projects.

**Comment by MarkBennett** [comment]
See #12074 for a similair request.
